main.go

```diff
@@ -175,15 +163,33 @@ func (p SSHCommandProvider) Execute(input *ExecuteInput) (*ExecuteResult, error)
 	}, nil
 }
 
+// LoadPrivateKeyFromConfig : Converts the PEM string into bytes and parse it
+func LoadPrivateKeyFromConfig(config SSHConfig) (ssh.Signer, error) {
```

Comment:
I'm wondering if these functions should be lower cased (loadPrivateKeyFromConfig and runCommand), since they won't be called from outside the file.